draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -623,8 +647,8 @@ resource is periodically distributed among the active nodes and where a node mig
 have to buffer data until an assigned transmission opportunity or when the physical path
 changes (e.g., when the length of a wireless path changes, or the physical layer changes
 its mode of operation).
-Variation also arises when a higher priority diffserv traffic classic prompts the
-transmission by a lower class. In these cases, the delay varies as a function of
+Variation also arises when traffic with a higher priority diffserv class pre-empts
+transmission of traffic with a lower class. In these cases, the delay varies as a function of
 external factors and attempting to infer congestion from an increase in the delay
 results in reduced throughput. The jitter from variation over short timescales
 might not be distinguishable similar from other effects.
```

Comment:
Not part of the PR, but the last sentence in that paragraph is weird. Do we want to say "It might be hard to distinguish the jitter due to variation over short timescales from other effects" ?